### PR TITLE
Ignore Unpublished Posts

### PR DESCRIPTION
**Pull request by pjlamb12**
Relates to #6 

Only add items to the feed if route.published or route.data.published are not explicitly set to false.

### CODE REVIEW

**Comment by pjlamb12** [comment]
@marcjulian @garygrossgarten the error on the CI is "Cannot find module feed". Any ideas why?

**Comment by garygrossgarten** [comment]
Hey there, does an `npm install` in the rss feed help?

**Comment by pjlamb12** [comment]
Doesn't the CI do the `npm install` by itself? I shouldn't need to do anything once CI kicks off for `npm install` to work, right?

**Comment by marcjulian** [comment]
@pjlamb12 the CI build should handle it, however, it was failing due to the plugins not being added to the root package.json
I fixed it now in master branch, please pull in the latest change from master.

Thanks for raising the issue with CI and thanks for sending this PR 😄

File: plugins/rss/src/rss.js [hunk 1]
```diff
@@ -17,31 +17,33 @@ const rssPlugin = async (html, route) => {
         feed.addCategory(cat);
       });
     }
-    const mdString = readFileSync(route.templateFile, 'utf8').toString();
+    if (route.published !== false || route.data.published !== false) {
```
Comment (marcjulian) [inline plugins/rss/src/rss.js:20]: I logged the route data and published is only available for `route.data.published`

```
{
  postRenderers: [ 'fouc', 'seoHrefOptimise', 'lazyImages', 'mediumZoom', 'rss' ],
  route: '/blog/2020-03-21-food',
  type: 'contentFolder',
  templateFile: '/Users/m/Dev/notiz/scully-plugins/scully-plugin-lazy-images/content/blog/2020-03-21-food.md',
  data: {
    name: undefined,
    title: 'Food',
    description: 'Food description',
    publishedAt: 2020-03-25T10:12:00.000Z,
    updatedAt: 2020-03-25T10:12:00.000Z,
    published: true,
    tags: [ 'lazy-images' ],
    authors: [ 'Bruce Lee' ],
    sourceFile: '2020-03-21-food.md'
  },
  config: {
    type: 'contentFolder',
    slug: { folder: './content/blog' },
    postRenderers: [ 'fouc', 'seoHrefOptimise', 'lazyImages', 'mediumZoom', 'rss' ]
  }
}
```

We can simplify the code with

```suggestion
    if (route.data.published) {
```

**Comment by pjlamb12** [comment]
@marcjulian thanks for the help! Got everything fixed and it's building now.

**Comment by marcjulian** [comment]
@pjlamb12 thank you!

**Pull request merged**